chore(compat_data): update compat-table to latest commit

Boshen opened the pull request:
Updates compat-table dependency to the latest commit.

- Previous SHA: `047878d9b67327ebd642ea855649e029fe066a50`
- New SHA: `a970fc00cc33b58d0b84d4b290ea46a185c8fcf1`

Dunqing commented:
### Merge activity

* **Sep 15, 2:33 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Boshen committed:
chore(compat_data): update compat-table to latest commit (#13761)

Updates compat-table dependency to the latest commit.

- Previous SHA: `047878d9b67327ebd642ea855649e029fe066a50`
- New SHA: `a970fc00cc33b58d0b84d4b290ea46a185c8fcf1`
